### PR TITLE
Add multiple faces option to impedance boundary

### PR DESCRIPTION
**Pull request by Samuelopez-ansys**
Add anisotropic impedance boundary

### CODE REVIEW

**Comment by Samuelopez-ansys** [comment]
@OscarCapac this Pull request replaces yours

File: pyaedt/hfss.py [hunk 1]
```diff
@@ -226,9 +226,18 @@ def field_setup_names(self):
...
+            PerfectH,
+            Aperture,
+            Radiation,
+            Impedance,
+            LayeredImp,
+            LumpedRLC,
+            FiniteCond,
+            Hybrid,
+            FEBI,
+        ) = range(0, 10)
```
Comment (SMoraisAnsys) [inline pyaedt/hfss.py:240]: Would it make sense / be possible to switch such implementation (and others) to `enum` ?

**Review by SMoraisAnsys — APPROVED** [review]
LGTM, good refact of some tests !

File: pyaedt/hfss.py (hunk 1 above)
Comment (Samuelopez-ansys) [reply pyaedt/hfss.py:240]: Yes, I will do it in a different PR

**Pull request merged**